src/components/shared/TokenInfoPopover/TokenInfo.tsx

```diff
@@ -20,86 +18,67 @@ interface Props {
 }
 
 const MSG = defineMessages({
-  nativeTokenMessage: {
-    id: `${displayName}.nativeTokenMessage`,
-    defaultMessage: "*This is the colony's native token",
+  nativeToken: {
+    id: `${displayName}.nativeToken`,
+    defaultMessage: 'Native',
   },
   viewOnEtherscan: {
     id: `${displayName}.viewOnEtherscan`,
     defaultMessage: 'View on {blockExplorerName}',
   },
-  addToWallet: {
-    id: `${displayName}.addToWallet`,
-    defaultMessage: 'Add token to Metamask',
-  },
 });
 
 const TokenInfo = ({ token, isTokenNative }: Props) => {
-  const { name, symbol, tokenAddress, decimals, thumbnail } = token;
-
-  const handleAddAssetToMetamask = () => {
-    // https://docs.metamask.io/wallet/how-to/register-token/
-    if (window.ethereum) {
-      window.ethereum
-        // @ts-ignore
-        .request({
-          method: RpcMethods.WatchAsset,
-          params: {
-            type: TokenType.Erc20,
-            options: {
-              address: tokenAddress,
-              symbol,
-              decimals,
-              image: thumbnail,
-            },
-          },
-        })
-        .catch((e) => {
-          console.error(e);
-        });
-    }
-  };
+  const { avatar, name, symbol, tokenAddress, thumbnail } = token;
+  const { formatMessage } = useIntl();
 
   return (
-    <div className={styles.main}>
-      <div className={styles.section}>
-        {name && (
-          <div title={name} className={styles.displayName}>
-            <TokenIcon token={token} size="xxs" />
-            {name}
+    <div className="flex flex-col items-center p-6 gap-6 w-80 text-gray-900">
+      <div className="flex flex-row items-center w-full gap-4">
+        <Avatar
+          size="m"
+          title={name}
+          avatar={thumbnail || avatar}
+          className="flex-shrink-0"
+        />
+        <div className="flex flex-col flex-1 gap-1 min-w-0">
+          <div className="w-full flex items-center">
+            <h4 className="heading-4 truncate">
+              {name} ({symbol})
+            </h4>
```

Comment:
![Screenshot from 2023-11-20 17-12-18](https://github.com/JoinColony/colonyCDapp/assets/1193222/79460b66-9a44-4b77-b768-6f51d0d1f8db)

Hovering this should show you the token's full name (via the in built browser tooltip)

Ie: set the `title` attribute